src/YoutubeAtomPlayer.tsx

```diff
@@ -32,6 +32,9 @@ declare global {
 
 type YoutubeCallback = (e: YT.PlayerEvent & YT.OnStateChangeEvent) => void;
 
+type PlayCustomEventDetail = { videoId: string };
+const customPlayEventName = 'video:play';
```

Comment:
And this just `customPlayEvent`?

Reply:
The thing is, this const is used for just the name of the event and not the actual event. That's why I thought maybe it's more clear if we call it `customPlayEventName` rather than `customPlayEvent`

But I'm happy to change if you think `customPlayEvent` suits better here?

Reply:
Yeah fair point, makes sense.